### ISSUE TITLE
Create a Calendar using Python

### ISSUE THREAD

**Issue by relaxedtomato** [nylas-samples/nylas-hacktoberfest-2022#36]
Code sample to create a calendar using Nylas Python SDK.

**Comment by SouvikDcoder** [comment]
Hey, @relaxedtomato I would like to contribute to this. Kindly assign the issue to me. Thanks!

**Comment by relaxedtomato** [comment]
@SouvikDcoder - let's tackle one issue at a time as separate PRs. I see you have started another issue, so will not assign this one for now.

**Comment by amit-ksh** [comment]
Can I take this issue?

**Comment by atejada** [comment]
@amit-ksh You already have this one https://github.com/nylas-samples/nylas-hacktoberfest-2022/issues/23 tackle it first before we can assign you a new one 😉

**Comment by relaxedtomato** [comment]
@atejada - just merged #23 😅

**Comment by Rkpani05** [comment]
can i try it ?

**Comment by atejada** [comment]
@Rkpani05 Sadly, @amit-ksh asked first, and his PR was merged, so I'm assigning him.
